packages/app-project/docker-compose-prod.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.4'
+
+services:
+  project-app:
+    image: front-end-monorepo_project-app:local
+    build:
+      context: .
+    command: [yarn, "dev"]
```

Comment:
Should this entry point be `yarn start`, since I can run a dev build with the other compose file?

Reply:
yeah - it could certainly be, i'll update.